### ISSUE TITLE
retina.js runs automatically despite using import or require

### ISSUE THREAD

**Issue by fahmilatib** [jgnewman/retinajs#280]
> To use retina.js the new-school way, you'll want to require it (or import it if you're using ES6) into your Gulp/Webpack/Grunt/CommonJS/etc application. In this case, the **script won't run automatically**. Instead, it'll let you determine when you'd like it to run. 

When I do `import 'retinajs'` or `require('retinajs')` the `window.retinajs` method runs automatically. Isn't it supposed to only run when I call it manually?

This contradicts what is written in the docs. Am I missing something here guys?

**Comment by strues** [comment]
Good catch @fahmilatib. It appears there's a bit of misinformation in the docs. I'll adjust it accordingly alongside an updated release, which should fix the issue.

**Comment by igolka97** [comment]
Any update on this?
